components/bookmark.css

```diff
@@ -0,0 +1,8 @@
+.card__button-bookmark {
+  background-color: #bea768;
+  padding: 20px;
+  border: 2px solid #e6664e;
+  position: absolute;
+  top: 155px;
+  right: 23px;
```

Comment:
To make the page more responsive it would be better to not use the absolute attribute and instead use %, em, rem, or vw and vh (viewport-width und height) but I also was wondering whether it is possible to do that with the bookmark because it "leaves" the parent-element...not sure